tests/trx_finality_status_test.py

```diff
@@ -150,16 +150,21 @@ def isState(state, expectedState, allowedState=None, notAllowedState=None):
     status.append(copy.copy(retStatus))
     startingBlockNum=testNode.getInfo()["head_block_num"]
 
-    def validateTrxState(status, present):
+    def validateTrxState(status):
+        Print(f"status: {status}")
+        present = True
+        state = getState(status)
+        if state == localState or state == unknownState:
+            present = False
```

Comment:
Combining knowledge from the fix in https://github.com/AntelopeIO/leap/pull/106, I'm wondering if, instead of removing the `present` argument, a determination needs to be done to calculate a correct `present` value for the first call to `validateTrxState` -- as it seems to be the only call that can have 2 expectations for the state. This way the test can continue to define its expectation of presence at each check.

Reply:
Would also recommend updating the Error message on line 149 to include both state options.